frontend/packages/dev-console/src/components/pipelines/pipeline-utils.ts

```diff
@@ -0,0 +1,21 @@
+import { k8sUpdate, K8sResourceKind, k8sGet } from '@console/internal/module/k8s';
+import * as _ from 'lodash';
+import { ServiceAccountModel } from '@console/internal/models';
+import { errorModal } from '@console/internal/components/modals';
+
+export const updateServiceAccount = (secret, serviceAccount): Promise<K8sResourceKind> => {
+  serviceAccount.secrets = [...serviceAccount.secrets, ...secret];
+  return k8sUpdate(ServiceAccountModel, serviceAccount);
+};
+
+export const associateServiceAccountToSecret = (secret, namespace) => {
+  k8sGet(ServiceAccountModel, 'pipeline', namespace)
```

Comment:
nit: move `pipeline` string to const variable

Reply:
Done